### PR TITLE
Fix analog joystick control

### PR DESCRIPTION
**Pull request by citelao**
This simple fix should fix a lot of the "drift" bugs and also adds a small movement dead zone to analog movements.

### CODE REVIEW

File: JSActionAnalog.m [hunk 1]
```diff
@@ -23,14 +23,18 @@ - (id) initWithIndex: (int)newIndex {
 }
 
 -(id) findSubActionForValue: (IOHIDValueRef) value {
+    int raw = IOHIDValueGetIntegerValue(value);
+    double parsed = [self getRealValue: raw];
+    
     if ([[subActions objectAtIndex: 2] active]) {
+        if (fabs(parsed) < 0.3) {
```
Comment (fyhuang) [inline JSActionAnalog.m]: Hey @citelao thanks for the PR! Could you pull out this `0.3` into a constant or something?

Comment (citelao) [inline JSActionAnalog.m]: Sure. Maybe put deadzones into the interface?

Comment (fyhuang) [inline JSActionAnalog.m]: Ha, if you're willing to do the work, that sounds great.

**Comment by citelao** [comment]
Should have those constants!

Configurability needs to wait for a later release.

**Comment by fyhuang** [comment]
Cool, looks good!

**Comment by andrewroth** [comment]
I'm still getting a lot of mouse drift to the top-left of the screen when using a wii nunchuck :(

**Comment by citelao** [comment]
@andrewroth as in you've bound two axes to move the mouse up and left and they're constantly being triggered?

**Comment by andrewroth** [comment]
The mouse is constantly being moved up and to the left, but I tried to set the axis to just horizontal and vertical movement, not just to move up and move left.  When I move the nunchuck to the bottom right it actually stops the mouse from moving up and to the left but doesn't move it down and to the right at all.

**Comment by citelao** [comment]
I don't think the released executable is the latest version.

Can you build from source? There are a few syntax errors with the latest version (`isa` is deprecated or something) but you can fix them with XCode's autosuggest.

**Comment by andrewroth** [comment]
I had a feeling that might be it.. but I did check the version in the about dialog and it matched the version numbers you have in git.  I'm happy to try a source build though.  Are there some instructions?

**Comment by citelao** [comment]
I didn’t change the version numbers :/

**Comment by andrewroth** [comment]
Hey, any hints for compiling it?  Do I just open in xcode and hit a compile now type button/menu item?

**Comment by citelao** [comment]
Yeah that's it. I have a compiled version but I can't upload it here... I'll put it on my forked version.

**Comment by citelao** [comment]
https://github.com/citelao/enjoy2/releases/tag/vidk

**Comment by andrewroth** [comment]
Much better!  It's usable.  It still drifts slowly to the left.  Oddly, even after I hit the Stop button it still drifts a bit... but I can use it.

**Pull request merged**